src/streamlink/plugins/kanal7.py

```diff
@@ -0,0 +1,33 @@
+"""
+$description Turkish live TV channel.
+$url kanal7.com
+$type live
+"""
+
+import logging
+import re
+
+from streamlink.plugin import Plugin, pluginmatcher
+from streamlink.plugin.api import validate
+from streamlink.stream.hls import HLSStream
+
+log = logging.getLogger(__name__)
+
+
+@pluginmatcher(re.compile(r"""
+    https?://(?:www\.)?
+    (?:
+        kanal7\.com/canli-izle
+    )
+""", re.VERBOSE))
```

Comment:
A verbose regex is unnecessary here, and so are the (non-capturing) parentheses.

```suggestion
@pluginmatcher(re.compile(
    r"https?://(?:www\.)?kanal7\.com/canli-izle",
))
```